pypsa/statistics.py

```diff
@@ -125,9 +144,9 @@ def __init__(self, network):
     def __call__(
         self,
         comps=None,
-        aggregate_time="mean",
```

Comment:
Does this need to be removed?

Reply:
I thought it is better because the call method is applied to all statistics methods. And as @Irieo suggested in his issue #578, this is confusing because one would assume that e.g. for curtailment you are not interested in the mean curtailment but in the total curtailment.